tangelo/helpers/utils.py

```diff
@@ -44,6 +44,22 @@ def assert_freq_dict_almost_equal(d1, d2, atol):
                 raise AssertionError(f"Frequency {k}, difference above tolerance {atol}: {d1[k]} != {d2[k]}")
 
 
+def assert_term_dict_almost_equal(d1, d2, delta=1e-6):
```

Comment:
We have a qubit operator class with built-in operations like `__eq__` etc
Does it make sense to create a method for `almost_equal`, that takes a tolerance (and ignores mapping)?

Rather than for people to end up finding this helper by luck, I'd instead extend our class to raise their chance of finding it. It is totally related to qubit operators.


You also could look instead at the absolute value of your highest coefficient of the terms dictionary appearing in the difference of the two qubit operators (difference in mappings may raise an error and prevent the operation).

Reply:
Thanks for the heads-up,, I will look into this.

Reply:
`__eq__` was hard to use, as every digit seems to be checked. I decided to use the `isclose` method instead, with a tight tolerance (https://quantumai.google/reference/python/openfermion/ops/SymbolicOperator#isclose).